app/components/cms_split_horizontal_card_component/cms_split_horizontal_card_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render GovGridRowComponent.new(additional_classes: wrapper_classes) do |row|  %>
+  <%= row.with_column("full") do %>
+    <% if @section_title %>
+      <h2 class="govuk-heading-m"><%= @section_title %></h2>
+    <% end %>
+      <div class="tc-row">
```

Comment:
I'm being super nit-picky here - then indentation on this div needs moving back.